<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(229, 188, 49);
font: italic 12pt &quot;Cambria&quot;;
background-color: rgb(33, 37, 41);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="InNumbersDollar">
      <property name="maximum">
       <double>1000000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="TauxChange">
      <property name="text">
       <string>Taux de Conversion : 1 euro = 1,05 Dollars Us</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="ValDollar">
      <property name="text">
       <string>Valeur en Dollars</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="Title">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 26pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string>Convertisseur</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="InNumbersEuro">
      <property name="maximum">
       <double>1000000000000000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="Btnconv">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 37, 41);
font: 700  14pt &quot;Cambria&quot;;</string>
      </property>
      <property name="text">
       <string>Convertir</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ValEuro">
      <property name="text">
       <string>Valeur en Euro</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionTaux_de_Change"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTaux_de_Change">
   <property name="text">
    <string>Taux de Change</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A Propos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
